lib/rules/no-get.js

```diff
@@ -80,6 +80,74 @@ function fixGet({
   return fixer.replaceText(node, `${objectTextSafe}${objectPathSeparator}${replacementPath}`);
 }
 
+function reportGetProperties({ context, node, objectText, properties }) {
+  let _properties = properties;
+  if (properties[0].type === 'ArrayExpression') {
+    // When properties are in an array(e.g. getProperties(this.obj, [bar, foo]) ), actual properties are under Array.elements.
+    _properties = properties[0].elements;
+  }
+  const propertyNames = _properties.map((arg) => arg.value);
+
+  context.report({
+    node,
+    message: ERROR_MESSAGE_GET_PROPERTIES,
+    fix(fixer) {
+      return fixGetProperties({
+        fixer,
+        node,
+        objectText,
+        propertyNames,
+      });
+    },
+  });
+}
+
+function fixGetProperties({ fixer, node, objectText, propertyNames }) {
+  if (!propertyNames.every((name) => isValidJSVariableName(name) || isValidJSArrayIndex(name))) {
+    // Do not autofix if any property is invalid.
+    return null;
+  }
+
+  if (node.parent.type === 'VariableDeclarator' && node.parent.id.type === 'ObjectPattern') {
+    // When destructuring assignment is in the left side of "=".
+    // Example: const { foo, bar } = getProperties(this.obj, "foo", "bar");
+    // Expectation:
+    // const foo = this.obj.foo;
+    // const bar = this.obj.bar;
+
+    if (node.parent.parent.type !== 'VariableDeclaration' && !node.parent.parent.kind) {
+      // It has to start with a declaration text. e.g. "const", "let"
+      return null;
```

Comment:
According to `yarn test --coverage`, this line is not covered by a test yet.

Reply:
no need to check this anymore with the new approach.